docs/products/opensearch/reference/index-replication.rst

```diff
@@ -0,0 +1,36 @@
+Automatic adjustment of replication factors
+===========================================
+
+Aiven for OpenSearch automatically adjusts index replication factors
+to ensure data availability and service functionality.
+
+When replication factor is automatically adjusted?
```

Comment:
It was like this in the original, but if I was writing this from scratch, I'd put the "`is`" after the "`When`" - that is, "`When is ...`"